pkgs/apps/exatensor/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, fetchFromGitLab, blas, mpi } :
+
+let
+  version = "73fef4";
```

Comment:
Please use the date of the commit as version number (formatted YYYY-MM-DD).

Reply:
Ok, thats a good point.